index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <h1>Hello Microverse</h1>
+</body>
```

Comment:
- [ ] According to the project requirement kindly consider using html file that includes a header with the text "Hello Microverse!". Use h1 tag for that for proper semantic usage.

Reply:
Please see inside the <body> tag i have used <h1>Hello Microverse!</h1> as per your requirement